### DISCUSSION TITLE
Removing lots of repositories in GrandOrgue

### DISCUSSION THREAD

**Discussion by oleg68** [GrandOrgue/grandorgue#1892] [Developers]
There are lot of repositories under GrandOrgue organisation:

https://github.com/GrandOrgue/YamlCppAdd
https://github.com/GrandOrgue/WxOsX
https://github.com/GrandOrgue/WavPackCross
https://github.com/GrandOrgue/PortaudioMingw
https://github.com/GrandOrgue/Mingw64LibGnuRx

Earlier they were used for building GrandOrgue for windows or for mac. But since #1854 we use the standard msys2 packet repository, so the repositories mentioned above are not more in use. So I'm going to remove them.

There is also https://github.com/GrandOrgue/WxWidgetsCross that is still used for `GoOdf`, so it should remain.

@larspalo @rousseldenis do you agree?

**Comment by larspalo** [comment]
@oleg68 Yes I agree, I'll eventually try to migrate the build of GoOdf to be done in a similar fashion, but for now it's perhaps best to leave the WxWidgetsCross.

**Comment by oleg68** [comment]
I removed the extra repos
